components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/mgwdefinition/MgwAPIDefinition.java

```diff
@@ -117,4 +119,22 @@ public String getResponseInterceptor() {
     public void setResponseInterceptor(String responseInterceptor) {
         this.responseInterceptor = responseInterceptor;
     }
+
+    /**
+     * Identify if the API information used or not
```

Comment:
Can you add `.` at the end of first sentence in the javadoc description. Make sure this is followed in the whole diff.
```suggestion
     * Identify if the API information used or not.
```